### PR TITLE
Add rendered stack

### PR DESCRIPTION
**Pull request by syncush**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -6,18 +6,20 @@ services:
       context: ./openstreetmap-website
       dockerfile: Dockerfile
     ports:
-      - "80:80"
+      - "8005:80"
```
Comment (syncush) [inline docker-compose.yml:9]: port should be 80

File: TODO.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+- tiler-renderer apache2 sometimes not running
```
Comment (syncush) [inline TODO.md:1]: isn't it fixed?

File: tiler-imposm/start.sh [hunk 3]
```diff
@@ -52,7 +52,7 @@ function updateData(){
             sleep 1m
         done
     else
-        imposm run -config config.json -cachedir $cachedir -diffdir $diffdir -limitto /mnt/data/$limitFile &
+        echo "fucker" & imposm run -config config.json -cachedir $cachedir -diffdir $diffdir -limitto /mnt/data/$limitFile &
```
Comment (syncush) [inline tiler-imposm/start.sh:55]: is the `echo "fucker"` needed?

File: tiler-renderer/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+ libxml2-dev \
+ libbz2-dev \
+ libgeos-dev \
+ libgeos++-dev \
+ libproj-dev \ 
+ gdal-bin \
+ libmapnik-dev \
+ mapnik-utils \
+ python-mapnik \
+ python3-psycopg2
```
Comment (syncush) [inline tiler-renderer/Dockerfile:50]: Can we merge these installs?

File: helm/osm-seed/values.yaml [hunk 5]
```diff
@@ -300,7 +300,7 @@ tilerServer:
     # AWS_ElasticBlockStore_volumeID : vol-1234bcd
     # AWS_ElasticBlockStore_size: 500Gi
     # In case cloudProvider: gcp
-    GCP_gcePersistentDisk_pdName: osmseed-disk-tiler-server-v2
+    GCP_gcePersistentDisk_pdName: osmseed-disk-tiler-server
     GCP_gcePersistentDisk_size: 50Gi
   resources:
     enabled: false
```
[the remark this answers is not in the document]
Comment (melancholiai) [reply helm/osm-seed/values.yaml:306]: done.

File: docker-compose.yml [hunk 6]
```diff
@@ -6,18 +6,20 @@ services:
...
       - db
+      - tiler-renderer
   db:
     image: osmseed-db:v1
     build:
       context: ./db
       dockerfile: Dockerfile
     ports:
-      - "5432:5432"
+      - "5435:5432"
```
[the remark this answers is not in the document]
Comment (melancholiai) [reply docker-compose.yml:22]: done.

File: openstreetmap-website/config/application.yml [hunk 7]
```diff
@@ -81,7 +82,7 @@ defaults: &defaults
   # OAuth consumer key for the web site
   #oauth_key: ""
   # OAuth consumer key for iD
-  #id_key: ""
+  id_key: "3oHU1PMG1DUEPKGaVevEwBufca51inKAEXhTcw03"
```
[the remark this answers is not in the document]
Comment (melancholiai) [reply openstreetmap-website/config/application.yml:85]: done.

File: docker-compose.yml (hunk 1 above)
Comment (melancholiai) [reply docker-compose.yml:9]: done.

File: TODO.md (hunk 2 above)
Comment (melancholiai) [reply TODO.md:1]: yep.

File: tiler-renderer/Dockerfile (hunk 4 above)
Comment (melancholiai) [reply tiler-renderer/Dockerfile:50]: done.

File: tiler-imposm/start.sh (hunk 3 above)
Comment (melancholiai) [reply tiler-imposm/start.sh:55]: surprisingly no.

**Review by syncush — COMMENTED** [review]
Approved

**Pull request merged**